setup.sh

```diff
@@ -26,38 +26,3 @@ sudo mkdir -p /var/cache/pbuilder/hook.d/
 cp configs/pbuilderrc ~/.pbuilderrc
 sudo cp configs/D70results /var/cache/pbuilder/hook.d/
 
```

Comment:
Why not add the fetching and running repo here?

Reply:
Thanks, sure, it's done now.